src/resolve/borders.ts

```diff
@@ -35,8 +35,22 @@ export function border(value: string, theme?: TwTheme): StyleIR | null {
       colorType = `borderRight`;
       break;
   }
+  const hasBorderColor = color(colorType, rest, theme?.borderColor);
+  if (hasBorderColor) {
+    return color(colorType, rest, theme?.borderColor);
```

Comment:
instead of recomputing this, let's just rename the variable `colorStyle`, check if it's truthy, and return it if it is.

```ts
  const colorStyle = color(colorType, rest, theme?.borderColor);
  if (colorStyle) {
    return colorStyle;
  }
```

Reply:
Sure 👍